src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
The only time I see us setting the base RIDs as win7-x64/osx.10.10-x64 is if we are always going to keep them around and if we are going to assert that assets for them will work on newer version of the platforms (which we cannot guarantee for OSX anyways). At best, we keep win7-x64 forever and fallback to it.

And even then, our story is busted on Linux.